src/internal/observe/observe.go

```diff
@@ -14,15 +14,16 @@ import (
 )
 
 const (
-	noProgressBarsFN = "no-progress-bars"
-	progressBarWidth = 32
+	noProgressBarsFN   = "no-progress-bars"
+	saveProgressBarsFN = "save-progress-bars"
```

Comment:
Just a tiny nit. Should we consider using the one of `{save,keep}{-,-completed-}progress-bars` instead of `save-progress-bars`? Save sounds like we are writing something to disk, could be just me.